ECHO-Sunrise.ipynb

```diff
@@ -385,71 +315,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "naics = echo_data.loc[(echo_data[\"FAC_NAICS_CODES\"].notna())] # We can only look at facilities that have at least one NAICS code\n",
-    "pdcjs = naics.loc[(naics[\"FAC_NAICS_CODES\"].str.contains(\"922140\"))]\n",
-    "mapper_marker(pdcjs)"
+    "if (district_program_data.shape[0] > 0):\n",
+    "    mp = mapper_area(ranked, geo_json_data, program.agg_col, program.unit)\n",
```

Comment:
Possible to add a legend, especially one that programmatically labels whether it's RCRA, CAA etc. and whether they are enforcements, violations, etc.? And explicitly shows that bigger circles = more. Figures should be standalone.

Even better if the colors are different based on the category, for easier comparison across various runs of the notebooks

Reply:
Yea, this is a huge issue! We definitely need a legend, but there are some stupid technical hurdles. There is no good plugin to do this, so we have to write a script from scratch (or find a good example to work from) that programmatically writes HTML that can be appended to the interactive map.

However, I do think adding a title is more short-term doable, as is changing colors based on category - great idea!

Reply:
added to #28